app_dart/lib/src/service/scheduler.dart

```diff
@@ -450,7 +450,8 @@ class Scheduler {
             success = true;
           }
         } else {
-          success = await luciBuildService.rescheduleUsingCheckRunEvent(checkRunEvent);
+          await luciBuildService.rescheduleUsingCheckRunEvent(checkRunEvent);
```

Comment:
Are you going to do anything with the new return object?

Reply:
Actually, I'm not sure. I'm addressing this issue as a first contribution and it doesn't mention that part: https://github.com/flutter/flutter/issues/83945

Reply:
Ah I see. Okay no problem.